SU-HSDO/suhumsci
# HSD8-596 Added plugin for clone action to change field values

# READY FOR REVIEW

# Summary
- Add to clone node action to allow for the user to incrementally increase date field values. Used for recurring events.

# Needed By (Date)
- 2/27

# Urgency
- High

# Steps to Test
1. Checkout this branch
1. `composer install --prefer-source` for good measure
1. sync a site `blt drupal:sync --site=[sitename]`
1. On `/admin/content` page, find an event (or make one if none exist) and click the checkbox for that row and choose "Clone" action at the top
1. in the next form, expand the "Adjust Cloned Field Values" and set the incremental values for the date field.
1. Choose to clone at least 3 times for best testing
1. click apply to clone the node
1. View the new nodes and validate each one has incremented date value. for example if the original event was 1/15/2019 12:01PM and you configured to increment by 5 hours. the cloned content should be 1/15/2019 5:01PM, 1/15/2019 10:01PM, 1/16/2019 3:01AM respectively.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
